packages/client/src/pages/ErrorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,84 @@
+import styled from 'styled-components';
+import Link from 'components/Link';
+import { breakpoints, paths } from '../../components/App/constants';
+import { H3 } from '../../design/H3';
+
+const { welcome } = paths;
+const { mobileM } = breakpoints;
+
+const ErrorPage = () => {
+  return (
+    <Page>
+      <Container>
+        <H3>
+          Что то пошло не так,
+          <br />
```

Comment:
Давай сделаем несколько текстовых элементов и отделим их марджинами
```suggestion
          <H3>Что то пошло не так,</H3> // сюда можно накинуть styled (styledH3) и добавить ему margin
          <H3>не переживай</H3>
```

Reply:
Сделал